### ISSUE TITLE
sqlite_processor error when on a hunt for column names

### ISSUE THREAD

**Issue by mashedkeyboard** [sdispater/orator#110]
Having some trouble trying to get a list of columns in a table for a class. The code I'm using is the following:
```
    for column in dbmgr.schema.connection().get_column_listing(model.get_table()):
        print(column)
```
However, all I get is an error from the sqlite_processor:
```
  File "/home/curtispf/PycharmProjects/tagtrak/tagtrak.py", line 26, in additem
    for column in dbmgr.schema.connection().get_column_listing(model.get_table()):
  File "/home/curtispf/.tagtrak-env/lib/python3.5/site-packages/orator/query/processors/sqlite_processor.py", line 18, in <lambda>
    return map(lambda x: x['column_name'], results)
  File "/home/curtispf/.tagtrak-env/lib/python3.5/site-packages/orator/connectors/sqlite_connector.py", line 34, in __getitem__
    return self.dict[item]
KeyError: 'column_name'
```
Any ideas why this might be? Not sure if this is a bug or me doing something wrong.

**Comment by sdispater** [comment]
It's a bug.

I will look into it and get back to you.

**Comment by mashedkeyboard** [comment]
Thanks, appreciated!

**Comment by sdispater** [comment]
I just pushed a fix on the `0.9` branch. You can test if it works for you.
